<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapSettings</class>
 <widget class="QWidget" name="MapSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lThemeId">
     <property name="text">
      <string>Вид карты:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbThemeId"/>
   </item>
   <item>
    <widget class="QLabel" name="lProjection">
     <property name="text">
      <string>Картографическая проекция:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbProjection"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0">
      <widget class="QCheckBox" name="cbShowAtmosphere">
       <property name="text">
        <string>Атмосфера</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="cbShowClowds">
       <property name="text">
        <string>Облака</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="cbShowRelief">
       <property name="text">
        <string>Рельеф</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="cbShowScaleBar">
       <property name="text">
        <string>Масштабная линейка</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Географические объекты</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QCheckBox" name="cbShowLakes">
       <property name="text">
        <string>Озера</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="cbShowBackground">
       <property name="text">
        <string>Фон</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="cbShowSubSolarPointIcon">
       <property name="text">
        <string>Подсолнечная точка</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QCheckBox" name="cbShowRivers">
       <property name="text">
        <string>Реки</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Окружение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="cbShowCompass">
       <property name="text">
        <string>Компас</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Компоненты управления</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="cbShowBorders">
       <property name="text">
        <string>Границы</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="cbShowCities">
       <property name="text">
        <string>Города</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QCheckBox" name="cbShowTerrain">
       <property name="text">
        <string>Местность</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="cbShowPlaces">
       <property name="text">
        <string>Места</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="cbShowCityLights">
       <property name="text">
        <string>Огни городов</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QCheckBox" name="cbShowIceLayer">
       <property name="text">
        <string>Ледовый покров</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="cbShowGrid">
       <property name="text">
        <string>Сетка</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="cbShowCrosshairs">
       <property name="text">
        <string>Прицел</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="cbShowSunShading">
       <property name="text">
        <string>Тень Солнца</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="cbShowOthePlaces">
       <property name="text">
        <string>Другие места</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="cbShowOverviewMap">
       <property name="text">
        <string>Обзорная карта</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="cbShowNavigationPanel">
       <property name="text">
        <string>Панель навигации</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
